src/app/containers/MostRead/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,591 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MostReadContainerCanonical renders most read as expected on canonical for news 1`] = `
+<div>
+  <section
+    aria-labelledby="Most-Read"
```

Comment:
Did we flag this in the accessibility swarm? `aria-labelledby` should be the ID of another element that provides a label for this element, I don't think this is quite being used correctly :)

Reply:
I think this looks fine @tochwill it maps to a span within the h2 which contains the title for this component 'Most read'

Reply:
Tested on Safari just now and it worked fine, realised the issue I brought up yesterday about the region not being read out properly was because I was using Chrome.

Reply:
Ah! `Most-Read` _is_ an ID. Whoops, my mistake.

Reply:
We did indeed flag it in the accessibility swarm, looks like it was also fixed!